src/map/packets.h

```diff
@@ -2299,6 +2299,16 @@ packet(0x96e,-1,clif->ackmergeitems);
 	// New Packets End
 #endif
 
+#if PACKETVER >= 20130320
+// new packets
+// changed packet sizes
+	packet(0x09a7,10,clif->pBankDeposit,2,4,6); // CZ_REQ_BANKING_DEPOSIT
```

Comment:
@Asheraf you enabled other functions what was disabled for this versions?
Because in code can be many #ifdefs related to this.
Or you mean banking in client not implemented?